Applications/Spire/Source/Ui/CalendarDatePicker.cpp

```diff
@@ -101,6 +133,49 @@ class MonthSpinner : public QWidget {
     }
 };
 
+class CalendarDayLabel : public QWidget {
+  public:
+    CalendarDayLabel(
+        date day, date::month_type month, QWidget* parent = nullptr)
+        : QWidget(parent) {
+      setFixedSize(scale(24, 24));
+      auto layout = new QHBoxLayout(this);
```

Comment:
Even here, jump from ```layout``` to ```label``` back to ```layout```, but ```layout``` can be worked with  after line 169 without any dependencies/issues.